extensions/youtube-downloader/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "youtube-downloader",
```

Comment:
@pernielsentikaer should we change the extension name internal name and folder name too?

Reply:
I don't think we need to do that right in this PR